helpers/formValidation.tsx

```diff
@@ -60,7 +60,10 @@ const signupValidation = Yup.object({
     .min(TEXT_MIN, `Must be at least ${TEXT_MIN} characters`)
     .max(TEXT_MAX, `Must be ${TEXT_MAX} characters or less`)
     .trim('Leading and trailing value must be alphanumeric character')
-    .required('Required')
+    .required('Required'),
```

Comment:
I notice lots of repeated strings in this file. There should be more constants made to be used in this file, rather than repeating the same strings over and over again

Reply:
Do you think it should be done in this PR or another PR?

Reply:
I think refactoring and feature should be done in different PRs, makes it easier to read